default-menu-auto-hide.ks.in

```
@@ -0,0 +1,17 @@
+#version=DEVEL
+#test name: default-menu-auto-hide
```

Shouldn't the test be called `default-menu-auto-hide-rhel`?

---

Fixed.